### ISSUE TITLE
Ligation site in configuration file

### ISSUE THREAD

**Issue by think-o** [nservant/HiC-Pro#477]
Hello!

I wanted to know how the ligation site is to be given in the configuration file. I suppose for HindIII since the cut site is A^AGCTT the reverse complement AAGCT is added to the 5' end of it. So the ligation site is AAGCTAGCTT as mentioned in the config file by default. 
 Can I do the same with the other enzymes by generating a reverse complement and add it to the 5' end to make it a ligation site.

**Comment by nservant** [comment]
Hi,
Yes in theory, but it might change according to the restriction enzyme and the overhang.
Which enzyme did you use ?
N

**Comment by think-o** [comment]
> Hi,
> Yes in theory, but it might change according to the restriction enzyme and the overhang.
> Which enzyme did you use ?
> N

Thanks for the response @nservant . I have a few HiC-data sets for the enzyme BglII and some other for DpnII. There are some for HindIII which I have run with default parameters as present in the config file

**Comment by nservant** [comment]
ok so for DpnII, the restriction motif if ^GATC and the ligation motif GATCGATC
And for Bglii, the restriction motif is A^GATCT and the ligation should be AGATCGATCT
Do you agree ?
N

**Comment by think-o** [comment]
Yeah. I was just wondering why the first base missed out while generating the ligation motif here
